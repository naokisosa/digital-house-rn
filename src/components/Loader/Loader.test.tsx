import 'react-native';
import React from 'react';

import Loader from './Loader';
import renderer from 'react-test-renderer';

describe('Test Loader component', () => {
  it('renders correctly', () => {
    const { toJSON } = renderer.create(<Loader />);
    expect(toJSON()).toMatchSnapshot();
  });
});
