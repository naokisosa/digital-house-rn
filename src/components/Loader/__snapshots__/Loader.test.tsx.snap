// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator />
</View>
`;
